core/app/[locale]/(default)/page.tsx

```diff
@@ -67,14 +67,12 @@ export default async function Home({ params }: Props) {
         title={t('FeaturedProducts.title')}
       />
 
-      <div className="my-10">
-        <ProductCardCarousel
-          products={newestProducts}
-          showCart={false}
-          showCompare={false}
-          title={t('Carousel.newestProducts')}
-        />
-      </div>
+      <FeaturedProductsCarousel
+        cta={{ label: t('NewestProducts.cta'), href: '/shop-all/?sort=newest' }}
+        description="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
```

Comment:
Can we update this to something that is not lorem ipsum?

Reply:
For sure, will ask for advice on this.